cli/module_registry.go

```diff
@@ -528,6 +528,11 @@ func validateModuleFile(client *viamClient, c *cli.Context, moduleID moduleID, t
 	if err != nil {
 		return err
 	}
+
+	if versionHasOnlyApps(getModuleResp.Module, version) {
+		return nil // no need to validate the module file for a module with only apps associated with
+	}
```

Comment:
this is the main change that allows us to upload modules with apps without the `--force` flag. currently skipping validation all together but in the future we could enforce that the for each app exists